Makefile

```diff
@@ -25,4 +27,8 @@ smoke: FORCE ## Runs smoke tests to verify the bundle install
 	@echo "Testing the new operator bundle install" 
 	@./scripts/test-bundle-install.sh
 
+.PHONY: container-build-index
+container-build-index:
```

Comment:
can you rename the target into: `index`  instead of `container-build-index`
and remove the .PHONY target